tools/peptideshaker/searchgui.xml

```diff
@@ -184,139 +346,534 @@
+    	    	<param name="omssa_hitlist_length" type="integer" value="0" 
+    	    		label="OMSSA: Hitlist Length" help="OMSSA hitlist length, 0 means all" />
+        		<param name="omssa_hitlist_charge" type="integer" value="30" 
+        			label="OMSSA: Number of Hits per Spectrum per Charge" help="number of hits per spectrum per charge" />
+	   		 	<param name="omssa_min_pep_length" type="integer" value="4" 
+	   		 		label="OMSSA: Minumum Peptide Length" help="Minimum length of peptides for no-enzyme and semi-tryptic searches" />
+	   	 		<param name="omssa_max_pep_length" type="integer" value="40" 
+	   	 			label="OMSSA: Maximum Peptide Length" help="Maximum length of peptides for no-enzyme and semi-tryptic searches (0: none)" />
+	   	 		<param name="omssa_format" label="OMSSA output format" type="select" >
+                	<option value="0" selected="True">OMX</option>
+                	<option value="1" >CSV</option>
+                </param> 
+				
+            </when>
+        </conditional>
 
-                <conditional name="msgf">
-                    <param name="msgf_selector" type="select" label="Run MSGF search">
-                        <option value="yes" selected="True">Search with MSGF</option>
-                        <option value="no">No MSGF search</option>
-                    </param>
-                    <when value="no" />
-                    <when value="yes">
-                        <param name="msgf_decoy" type="boolean" truevalue="1" falsevalue="0"
-                            label="Search Decoys" help="If yes then a decoy database will be generated and searched. Assumed input database contains no decoys"/>
-                        <param name="msgf_min_pep_length" type="integer" value="6"
-                            label="Minimum Peptide Length" help="Minimum length for a peptide to be considered"/>
-                        <param name="msgf_max_pep_length" type="integer" value="30" 
-                            label="Maximum Peptide Length" help="Maximum length for a peptide to be considered"/>
-                        <param name="msgf_termini" type="select" format="text" 
-                            label="Number of tolerable termini" help="Searches will take much longer if selecting a value other than 2">
-                            <option value="0">0 (ie non-specific cleavage)</option>
-                            <option value="1">1 (ie semi-tryptic cleavage)</option>
-                            <option value="2" selected="true">2 (ie fully-tryptic cleavage)</option>
-                        </param>
-                        <param name="msgf_num_ptms" label="Max PTMs per peptide" type="integer" value="2"/>
-                    </when>
-                </conditional>
+		<!-- MS-GF+ ADVANCED PARAMETERS -->
+        <conditional name="msgf">
+            <param name="msgf_advanced" type="select" label="MSGF Options">
```

Comment:
The tool has been bumped up in version number.  I suspect there are so many things in this change that would affect backwards compatibility that we need to accept workflow breakages anyway